website/client/js/controllers/inventoryCtrl.js

```diff
@@ -54,11 +54,16 @@ habitrpg.controller("InventoryCtrl",
 
     $scope.$watch('user.items.gear', function(gear){
       $scope.gear = {};
+      $scope.gearByType = {};
       _.each(gear.owned, function(v,key){
         if (v === false) return;
         var item = Content.gear.flat[key];
-        if (!$scope.gear[item.klass]) $scope.gear[item.klass] = [];
+        if (!$scope.gear[item.klass])
```

Comment:
You should put brackets around the consequences of the `if` block, even if it's just one line.  That way, if more logic is added at a later date, it won't be accidentally outside of the logic block.

Still though, not everyone cares about this guideline.  I'm not sure what the rules are for this repo.